api/swagger.yaml

```diff
@@ -8339,6 +8339,8 @@ paths:
                   UsageData:
                     Size: 10920104
                     RefCount: 2
+              BuildCache: null
+              BuilderSize: 0
```

Comment:
Arf.. this one lead me down another rabbit-hole. I noticed the `BuilderSize` field is marked deprecated.

Digging a bit in history, the `BuilderSize` field:

- Was _added_ in https://github.com/moby/moby/commit/5c3d2d552b0430672d5f481ab2d37036f6e92166 (https://github.com/moby/moby/pull/32677)
    - this went into docker 17.07 (API v1.31)
    - implemented as "experimental" feature, and (hence?) not documented.
- The _cli_ did not start using this field until Docker 18.06, which added it to `docker system df` in https://github.com/docker/cli/commit/b95638a5acbecbb4b8fb0134a13089e4993df626 (https://github.com/docker/cli/pull/231); still experimental at that time.
- Was deprecated in docker 18.06 (API 1.38) through https://github.com/moby/moby/commit/b225258496f3ec8b6906b3fc6a3a2c76e50502d9 (https://github.com/moby/moby/pull/37151)
- The CLI was updated in https://github.com/docker/cli/commit/ca608c23027b6e82c4f6c63a53de738049b68d26 (docker 19.03) to no longer depend on the API field
    - older versions of the CLI depend on the field to present the build-context cache-size (but before that it was experimental, as mentioned above)

From that, I think we should:

- Not add it here (as: deprecated, and wasn't documented)
- Add `json:",omitempty"` on the field, and update the deprecation message, e.g:
    ```go
    // Deprecated: deprecated in API 1.38, and no longer used since API 1.40.
    ```
- Don't set the field in API >= v1.42
- Add an entry to the v1.42 changelog (docs/api/version-history.md), with something like:

    ```markdown
    * Removed the `BuildCache` field on the `GET /system/df` endpoint. This field
      was introduced in API 1.31 as part of an experimental feature, and no longer
      used since API 1.40.
    ```